cirq-core/cirq/ops/linear_combinations.py

```diff
@@ -498,6 +498,23 @@ def _unitary_(self) -> np.ndarray:
             return m
         raise ValueError(f'{self} is not unitary')
 
+    def _json_dict_(self):
+        def key_json(k):
+            # k is a frozenset, each element of frozen set is a tuple.
+            return [[x for x in e] for e in sorted(list(k))]
+
+        return {
+            'keys': [key_json(k) for k in self._linear_dict.keys()],
+            'values': list(self._linear_dict.values()),
```

Comment:
elsewhere, we serialize dictionaries (and dict-like things) with

```python
{'items': list(linear_dict.items())}
```

Reply:
done